core/src/messages.cpp

```diff
@@ -212,15 +212,10 @@ void to_json(nlohmann::json& doc, const manifestdoc& m) noexcept (false) {
     doc["attributes"]   = m.attr;
 }
 
-void to_json(nlohmann::json& doc, const basic_query& query) noexcept (false) {
-    doc["pid"]              = query.pid;
-    doc["token"]            = query.token;
-    doc["url-query"]        = query.url_query;
-    doc["guid"]             = query.guid;
-    doc["manifest"]         = query.manifest;
-    doc["storage_endpoint"] = query.storage_endpoint;
-    doc["function"]         = query.function;
-    doc["attributes"]       = query.attributes;
+void to_json(nlohmann::json& doc, const basic_query& query) noexcept(false) {
```

Comment:
I believe it would be better to simply remove the function

Reply:
Won't work because we explicitly make query things packable
https://github.com/equinor/oneseismic/blob/80b92a1ba1e651ee1a95bedd10766a91e9045f6a/core/src/messages.cpp#L559

So if the method is not present, compiler complains
```
...oneseismic/core/src/messages.cpp:590:17:   required from here
...oneseismic/core/src/messages.cpp:25:22: error: no matching function for call to ‘nlohmann::basic_json<>::basic_json(const one::slice_query&)’
     return nlohmann::json(self).dump();
```

Reply:
Alright, I would have liked for this to be caught at compile time, but I guess that is not possible to achieve. 

I suggest making the error message a bit more generic, such as "Packing is not implemented for query". This also points out that the operation is simply not implemented, advising what needs to be done if this operation should be needed later for whatever reason.